packages/ui-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -18,25 +18,31 @@ export type Props = {
 
 const SearchBar: React.FC<Props> = ({ query, onQueryChange, onClearButtonClick, onClose, inputRef }: Props) => {
   const { t } = useTranslation('search');
+  const handleSubmit = (event: React.ChangeEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    inputRef?.current?.blur();
+  };
 
   return (
     <div className={styles.searchBar}>
       <Icon icon={Search} className={styles.icon} />
-      <input
-        className={styles.input}
-        type="search"
-        value={query}
-        onChange={onQueryChange}
-        onKeyDown={(event) => event.key === 'Escape' && onClose?.()}
-        aria-label={t('search_bar.search_label')}
-        placeholder={t('search_bar.search_placeholder')}
-        ref={inputRef}
-      />
-      {query ? (
-        <IconButton className={styles.clearButton} aria-label={t('search_bar.clear_search_label')} onClick={onClearButtonClick}>
-          <Icon icon={Cancel} />
-        </IconButton>
-      ) : null}
+      <form className={styles.searchForm} role="search" onSubmit={handleSubmit}>
+        <input
```

Comment:
While we are optimising the searchbar even further than the (android) onscreen keyboard, lets also take one extra step to create a hidden label for the search bar. That is preferred above `aria-label` on the input. So that should be removed.
```suggestion
        <label for="searchbar-input className="hidden">{t('search_bar.search_label')}</label>
        <input
                id="searchbar-input"
                // etc
        >
```

Reply:
Thanks for this input! Do you mean to apply this label and give it the `hideUntilFocus` selector?

Reply:
No intentionally the `.hidden` class - which hides the element for normal users! It should not be shown when it's focussed, that has no use for a label.

(i find a classname such as `.srOnly` more descriptive - where sr means screenreader but thats not something for now!)

Reply:
Thanks for the clarification, I'll use the `.hidden` class